_data/components/grid.yml

```diff
@@ -77,6 +77,22 @@ attributes: [{
   name: "refresh-button"
 },{
   name: "quick-filter-columns"
+},{
+  name: "insert-button",
+  default: "false"
+},{
+  name: "insert-button-position",
+  type: "'bottom' | 'top'",
+  default: "bottom",
+  required: "",
+  description: "Indicates whether or not the insert button is attached at the top or at the bottom of the grid when the button is floatable"
```

Comment:
Here the clause "whether or not" is redundant
"Indicates the insert button is attached at the top or at the bottom of the grid when the button is floatable"